packages/router/__tests__/router-test.ts

```diff
@@ -98,119 +98,6 @@ afterEach(() => {
 
 describe("a router", () => {
   describe("init", () => {
-    it("initial state w/o hydrationData", async () => {
```

Comment:
Moved these tests to the new file above that deals strictly with router initialization/fallback tests